driveruipanels.h

```diff
@@ -107,7 +113,13 @@ class debug_panel : public ui_panel {
         m_scenariolines,
         m_eventqueuelines,
         m_powergridlines,
+ #ifdef WITH_UART
+        m_rendererlines,
```

Comment:
Brzydko tak wsadzać pod ifdefa niezwiązane parametry. W ogóle można to zostawić na stałe. Wyłącznik WITH_UART jest żeby dało się kompilować bez libserialport, jak zostanie kilka nieużywanych stringów to nic się nie stanie.

Reply:
ok